common/src/main/java/com/railwayteam/railways/mixin/MixinTrackMaterial.java

```diff
@@ -21,14 +21,18 @@ public class MixinTrackMaterial {
      */
     @Inject(method = "deserialize", at = @At("HEAD"), cancellable = true)
     private static void snr$updateCherryCompatTracks(String serializedName, CallbackInfoReturnable<TrackMaterial> cir) {
-        List<String> cherryList = List.of("biomesoplenty:cherry", "byg:cherry", "blue_skies:cherry");
-
-        if (cherryList.contains(serializedName))
-            cir.setReturnValue(CRTrackMaterials.CHERRY);
+        switch (serializedName) {
+            case "biomesoplenty:cherry", "byg:cherry", "blue_skies:cherry"
+                    -> cir.setReturnValue(CRTrackMaterials.CHERRY);
+            case "biomesoplenty:cherry_wide", "byg:cherry_wide", "blue_skies:cherry_wide"
+                    -> cir.setReturnValue(CRTrackMaterials.WIDE_GAUGE.get(CRTrackMaterials.CHERRY));
```

Comment:
this works, but it's better to use the helper function `CRTrackMaterials.getWide(<base>)`

Reply:
Fixed!